servers/rendering/renderer_rd/shaders/environment/sky.glsl

```diff
@@ -254,6 +256,9 @@ void main() {
 
 	frag_color.rgb = color;
 	frag_color.a = alpha;
+	
```

Comment:
```suggestion

```

The style check is failing because there is a tab in this line. You will have to delete it. Then, instead of making a new commit, use `git commit --amend` to fix up your existing commit, then force push to this branch

Reply:
Ah yep, had this on another commit as well (my IDE extensions use `clang-format` for C++ but seemingly not for GLSL, so I had been saving without formatting). Got `pre-commit` setup now so hopefully shouldn't be an issue moving forward.

Updated :)